Formula/newsboat.rb

```diff
@@ -19,6 +14,7 @@ class Newsboat < Formula
   needs :cxx11
 
   def install
+    ENV["XML_CATALOG_FILES"] = "/usr/local/etc/xml/catalog"
```

Comment:
```
    ENV["XML_CATALOG_FILES"] = etc/"xml/catalog" if build.head?
```

Reply:
Same as previous comment, not sure it's for head builds only